examples/Examples/Common/ExampleOptions.cs

```diff
@@ -0,0 +1,52 @@
+using CommandLine;
+
+namespace Concordium.Sdk.Examples.Common;
+
+/// <summary>
+/// Command line options for the runnable SDK examples.
+///
+/// This class can be used to specify options for the
+/// <see cref="Client.ConcordiumClient"/>
+/// at the command-line.
+/// </summary>
+public class ExampleOptions
+{
+    /// <summary>
+    /// URL representing the endpoint where the gRPC V2 API is served.
+    /// </summary>
+    public const string DefaultEndpoint = "https://localhost/";
+
+    /// <summary>
+    /// Default port at the endpoint where the gRPC V2 API is served.
+    /// </summary>
+    public const ushort DefaultPort = 20000;
+
+    /// <summary>
+    /// Default connection timeout in seconds.
+    /// </summary>
+    public const ushort DefaultTimeout = 60;
+
+    [Option(
+        'e',
+        "endpoint",
+        HelpText = "URL representing the endpoint where the gRPC V2 API is served.",
+        Default = DefaultEndpoint
+    )]
+    public string Endpoint { get; set; } = DefaultEndpoint;
```

Comment:
I dont’ think `.. = DefaultEndpoint;`

is used. It only uses the default if it is written in the attribute header. This is general throughout all options defined.

Reply:
I am not sure I understand 🤔 for instance if I do not specify a port with the `-p` option, it will use the value 20000 when this is specified for the default value. For instance, I get
```
$ ./Concordium.Sdk.Examples.Transactions.Transfer  -a 100 -r 4t6JVmGmsRTNDW4Rqw6KKh5NRucNgDyi8v7Rh19guBWLteEJVu -k ~/Desktop/stagenet-7.json -e https://service.internal.stagenet.concordium.com
Successfully submitted transfer transaction with hash 187ec51672a09460cdc844a61463f7039b3bebb5c0748208f2178ecbe5b89360
```
but
```
$ ./Concordium.Sdk.Examples.Transactions.Transfer  -a 100 -r 4t6JVmGmsRTNDW4Rqw6KKh5NRucNgDyi8v7Rh19guBWLteEJVu -k ~/Desktop/stagenet-7.json -e https://service.internal.stagenet.concordium.com -p 9000
```
hangs times out since connections on port 9000 are not accepted. Should this not be the case, and does it happen due to some coincidence?

Reply:
Ah, this was just to squelch the compiler. But I guess I can add a pragma...